docs/tutorials/platform_beginner/7_customize_login_and_register_forms.md

```diff
@@ -1,11 +1,55 @@
-# Step 7 - Customize the User forms
+# Step 7 - Customizing the user forms
 
-We want to have a consistent design over our web site. When using the pagelayout for the user forms, you see the design of your website. We want to welcome our new users with a pretty register page.
+We want to have a consistent design over our web site. When using the pagelayout for the register forms, you see the design of your website. We want to welcome our new users with a pretty register page.
 
-## Use custom template for user forms
 
+We want to have a consistent design over our web site. When using the register form, it was a raw HTML form without any styling on it. We want to welcome our new users with a pretty register page.
 
-At the same level than `pagelayout` add a `user_registration` directive so the beginning of your `views.yml` file looks like this:
+### Use a views.yml file
+
+In order to have a better file structure, we will extract the YAML keys regarding the presentation and put them into their own separate file. Create a  `views.yml` file and move the following code referring to templating to this file from `ezplatform.yml`:
+
+```yaml
+ezpublish:
+    system:
+        default:
```

Comment:
This should not be `default` anymore in v2, rather `site_group` for instance.

Otherwise the override rules will affect rendering of admin UI.